specification/src/processor-table.md

```diff
@@ -1,29 +1,66 @@
+- Base column `invu`, the list of inverses of unique clock jump differences, *i.e.*, without counting multiplicities. This column helps to select the unique nonzero `cjd`'s.
+- Extension column `rer`, the running evaluation of relevant clock cycles.
+- Extension column `reu`, the running evaluation of unique clock cycle differences.
+- Extension column `rpm`, the running product of all clock jump differences, with multiplicities.
 
- - `cjd`, the list of all clock jump differences greater than 1 in all memory-like tables.
- - `invm`, the list of inverses of clock jump differences, counting multiplicities. This column helps to select all nonzero `cjd`'s.
- - `invu`, the list of inverses of unique clock jump differences, *i.e.*, without counting multiplicities. This column helps to select the unique nonzero `cjd`'s.
+# Arithmetic Intermediate Representation
 
-… and three extension columns:
+Let all household items (🪥, 🛁, etc.) be challenges, concretely evaluation points, supplied by the verifier.
+Let all fruit & vegetables (🥝, 🥥, etc.) be challenges, concretely weights to compress rows, supplied by the verifier.
+Both types of challenges are X-field elements, _i.e._, elements of $\mathbb{F}_{p^3}$.
 
- - `rer`, the running evaluation of relevant clock cycles.
- - `reu`, the running evaluation of unique clock cycle differences.
- - `rpm`, the running product of all clock jump differences, with multiplicities.
+Note, that the transition constraint's use of `some_column` vs `some_column_next` might be a little unintuitive.
+For example, take the following part of some execution trace.
+
+| Clock Cycle | Current Instruction |  st0 |  …  | st15 | Running Evaluation “To Hash Table”  | Running Evaluation “From Hash Table”    |
+|:------------|:--------------------|-----:|:---:|-----:|:------------------------------------|:----------------------------------------|
+| $i-1$       | `foo`               |   17 |  …  |   22 | $a$                                 | $b$                                     |
+| $i$         | hash                |   17 |  …  |   22 | $🪣·a + \sum_j 🧄_j \cdot st_j$       | $b$                                     |
+| $i+1$       | `bar`               | 1337 |  …  |   22 | $🪣·a + \sum_{j=0}^9 🧄_j \cdot st_j$ | $🪟·b + \sum_{j=0}^4 🫑_j \cdot st_{j+5}$ |
+
+In order to verify the correctness of `RunningEvaluationToHashTable`, the corresponding transition constraint needs to conditionally “activate” on row-tuple ($i-1$, $i$), where it is conditional on `ci_next` (not `ci`), and verifies absorption of the next row, _i.e._, row $i$.
+However, in order to verify the correctness of `RunningEvaluationFromHashTable`, the corresponding transition constraint needs to conditionally “activate” on row-tuple ($i$, $i+1$), where it is conditional on `ci` (not `ci_next`), and verifies absorption of the next row, _i.e._, row $i+1$.
```

Comment:
@aszepieniec, how about this instead of a picture for explaining the potentially confusing use of `some_column` vs `some_column_next`?

Reply:
With the text explanation it is perfectly clear. (But the table cannot stand on its own.)